topics/introduction/tutorials/galaxy-intro-strands/tutorial.md

```diff
@@ -389,7 +389,7 @@ Galaxy excels at answering questions about genomic intervals and different sets
 
 > ### {% icon hands_on %} Hands-on: Genomic Interval Tools
 >
-> * In the tool panel, *open* the **Operate on Genomic Intervals** toolbox.  It's typically past the **NGS** toolboxes.
+> * In the tool panel, *open* the **Operate on Genomic Intervals** toolbox.  It's typically in the **Common Genomics Tools or Text Manipulation** sections.
```

Comment:
.org .eu and .org.au all have it under "Common Genomics Tools". Alternatively, I'd have users search for "interval".

Reply:
While the "Operate on Genomic Intervals" is in "Common Genomics Tools" on usegalaxy.eu it does not contain the Intersect tool:
![image](https://user-images.githubusercontent.com/4154788/96872880-79f82700-1474-11eb-95c6-4e3448b04fb1.png)

The procedure in this section encourages users to search through tool panel sections for useful looking tools. I am not sure if this is a good way to explore Galaxy given how many tools there are to search through.

Reply:
> While the "Operate on Genomic Intervals" is in "Common Genomics Tools" on usegalaxy.eu it does not contain the Intersect tool:

Ah, well spotted, it seems a bit weird that the GOPS tools are scattered around 3 sections ("Subtract" is under "Join, Subtract and Group") on UseGalaxy.eu .

> The procedure in this section encourages users to search through tool panel sections for useful looking tools. I am not sure if this is a good way to explore Galaxy given how many tools there are to search through. I'm trying to not make too many changes to the tutorial though.

Indeed, maybe having users search for "interval" (as I was suggesting as alternative) could be an instance-agnostic solution and not a very big change?

Reply:
Yes, good suggestion. I have updated the text accordingly.

Reply:
Thanks!